### DISCUSSION TITLE
dde-control-center的-p参数怎么用？

### DISCUSSION THREAD

**Discussion by iokala** [linuxdeepin/developer-center#2836] [Q&A | 问答板块]
有谁用过吗？举个例子

**Comment by justforlxz** [comment]
这个参数是用于内部调试的，可以直接显示到某个页面。

**Reply by QQLazy** [reply]
有具体案例不，不了解<page> 要输入什么。感谢。

**Reply by BLumia** [reply]
cc @robertkill

**Comment by Wing-summer** [comment]
你可以看我给你的回帖，写啥怎么查我都给你写指引了 :smile:

**Reply by BLumia** [reply]
可以链接到这儿或者直接贴过来，这样如果从搜索引擎查过来先搜到 GitHub 的话也可以顺着找过去/有所参考 :)

**Reply by Wing-summer** [reply]
相关帖子：https://bbs.deepin.org/post/245625

**Reply by Wing-summer** [reply]
> 可以链接到这儿或者直接贴过来，这样如果从搜索引擎查过来先搜到 GitHub 的话也可以顺着找过去/有所参考 :)

链接已贴到回复

**Reply by QQLazy** [reply]
感谢，参考后已经可以使用，O(∩_∩)O谢谢~
